pymilvus/client/abstract.py

```diff
@@ -652,7 +652,7 @@ class Hits(list):
     def __init__(
         self,
         topk: int,
-        pks: Union[int, str],
+        pks: List[Union[int, str]],
```

Comment:
self.ids is `List[Union[str, int]]`, so pks should be `List[Union[str, int]]` too.